<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QLabel" name="lbStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="loginFormGB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
    </font>
   </property>
   <property name="title">
    <string>Forma Login</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="userTB">
        <property name="font">
         <font>
          <family>Tahoma</family>
         </font>
        </property>
        <property name="placeholderText">
         <string>Përdoruesi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passTB">
        <property name="font">
         <font>
          <family>Tahoma</family>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Fjalëkalimi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="saveLogin">
        <property name="font">
         <font>
          <family>Tahoma</family>
         </font>
        </property>
        <property name="text">
         <string>Ruaj të dhënat</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loginBTN">
        <property name="font">
         <font>
          <family>Tahoma</family>
         </font>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fshiBTN">
        <property name="font">
         <font>
          <family>Tahoma</family>
         </font>
        </property>
        <property name="text">
         <string>Fshi formën</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="errLB">
      <property name="font">
       <font>
        <family>Tahoma</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="sessionsGB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
    </font>
   </property>
   <property name="title">
    <string>Sesione të ruajtura</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QListView" name="listSessions">
      <property name="font">
       <font>
        <family>Tahoma</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
